angular.json

```diff
@@ -94,5 +94,8 @@
         }
       }
     }
+  },
+  "cli": {
+    "analytics": false
```

Comment:
Not sure what this change is, but it looks fine

Reply:
This will stop  Angular asking permission to collect data

Reply:
This means we deny Angular permission to collect data